### PR TITLE
Add Adaptive to Noise Control and Conversation Awareness toggle.

### PR DESCRIPTION
**Pull request by hllvc**
## Description

- Added option to switch between `Adaptive` under Noise Control
- Added command to toggle `Conversation Awareness`

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/airpods-noise-control/package.json [hunk 1]
```diff
@@ -2,28 +2,110 @@
...
+  ],
   "categories": [
     "Media",
     "System"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "switch-noise-control",
```
Comment (pernielsentikaer) [inline extensions/airpods-noise-control/package.json:18]: Would be better to keep the same index if a command is the same, this way people don't lose a hotkey or ranking

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

- Could you fix the CI errors

- Add a [CHANGELOG](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) entry

@chrahe do you mind checking this too?

File: extensions/airpods-noise-control/src/airpods-menu.ts [hunk 2]
```diff
@@ -61,35 +83,55 @@ tell application "System Events"
...
+      return null;
+    }
+    case "airpods-not-connected": {
+      showFailureToast("", { title: "AirPods not connected!" });
+      return null;
+    }
+    default: {
+      return res;
+    }
+  }
```
Comment (hllvc) [inline extensions/airpods-noise-control/src/airpods-menu.ts:136]: @pernielsentikaer  There is some issue with `switch` indentation. When running `ray lint --fix` it sets up correct indentation for `switch`. But, when running only `ray lint` it shows the same error as CI.

Comment (pernielsentikaer) [reply extensions/airpods-noise-control/src/airpods-menu.ts:136]: Should be better now

**Comment by pernielsentikaer** [comment]
Then we just awaits @chrahe 🙂

**Comment by chrahe** [comment]
Hey,

so sorry, I've been very busy with personal matters so I haven't really been able to keep up with this issue & PR. It looks good to me, thanks for taking on the extension update :)

**Review by chrahe — APPROVED** [review]

**Comment by pernielsentikaer** [comment]
It's so fine @chrahe - life is more important after all 🙂

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**